devcareer/code-jammers-backend
# #174566179 Prepare app for heroku deployment - [staging]

### What does this PR do?

Creates a heroku environment for the app

### Description of Task to be completed?

- create procfile for use on heroku
- create build script to build app on local environment

### How should this be manually tested?

Scroll down to the deployment section of this PR and and click view deployment. It should send you to the review app URL

### What are the relevant pivotal tracker stories?

[#174566179](https://www.pivotaltracker.com/story/show/174566179)
